cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -188,6 +188,9 @@ private Channel connectViaCliPort(URL jenkins, CliPort clip) throws IOException
                 rsp.write(ch);
             }
             String head = new BufferedReader(new StringReader(rsp.toString("ISO-8859-1"))).readLine();
+            if (head == null) {
+                throw new IOException("Unexpected empty response");
```

Comment:
Clarify from what. According to comment before `Proxy response body must be not empty`